### PR TITLE
dptp-controller-manager: a new flag additional-image-stream-namespaces

### PR DESCRIPTION
**Pull request by hongkailiu**
https://github.com/openshift/release/pull/10475#issuecomment-664429816

/cc @alvaroaleman

### CODE REVIEW

File: pkg/controller/test-images-distributor/test_images_distributor.go [hunk 1]
```diff
@@ -430,7 +431,10 @@ func testInputImageStreamTagFilterFactory(l *logrus.Entry, ca agents.ConfigAgent
 			l.WithField("name", nn.String()).WithError(err).Error("Failed to get imagestreamname for imagestreamtag")
 			return false
 		}
-		if additionalImageStrams.Has(imageStreamName.String()) {
+		if additionalImageStreams.Has(imageStreamName.String()) {
+			return true
+		}
+		if additionalImageStreamNamespaces.Has(nn.Namespace) {
```
Comment (alvaroaleman) [inline pkg/controller/test-images-distributor/test_images_distributor.go:437]: We can move this above to line 20, the imageStream namespace and imagestreamTag namespace is guaranteed to be equal so we don't have to do this cheap check after using the indexer

Comment (hongkailiu) [reply pkg/controller/test-images-distributor/test_images_distributor.go]: `line 20`: 420?

Comment (alvaroaleman) [reply pkg/controller/test-images-distributor/test_images_distributor.go]: err yeah, thats what I meant

File: pkg/controller/test-images-distributor/test_images_distributor.go [hunk 2]
```diff
@@ -430,7 +434,7 @@ func testInputImageStreamTagFilterFactory(l *logrus.Entry, ca agents.ConfigAgent
 			l.WithField("name", nn.String()).WithError(err).Error("Failed to get imagestreamname for imagestreamtag")
 			return false
 		}
-		if additionalImageStrams.Has(imageStreamName.String()) {
+		if additionalImageStreams.Has(imageStreamName.String()) {
```
Comment (alvaroaleman) [inline pkg/controller/test-images-distributor/test_images_distributor.go:437]: My favorite typo it seems :D

**Pull request merged**